starlette_web/common/http/renderers.py

```diff
@@ -0,0 +1,39 @@
+import json
+from typing import Any
+
+from starlette.responses import Response, BackgroundTask
+
+
+class BaseRenderer(Response):
+    pass
+
+
+class JSONRenderer(BaseRenderer):
+    # TODO: encode with custom JSONEncoder (imported from Django)
+    json_encode_class = json.JSONEncoder
+    media_type = "application/json"
+
+    # Copied from starlette.responses.JSONResponse
```

Comment:
why? Can we inherit from `starlette.responses.JSONResponse` ?

Reply:
This is pointless, since we'll have to overload method `render` either way (it uses plain json.dumps without propagating encoder class)